### ISSUE TITLE
Pylsl time correction example

### ISSUE THREAD

**Issue by Mome** [sccn/labstreaminglayer#308]
It took me ages to  realized, that I am supposed to do a time correction every time I record from different computers. It would have helped, if there were python examples including the time correction. I could make a pull request including an such example. Are you interested?

**Comment by cboulay** [comment]
Yes. I'd like to know more about the problem/confusion you faced but maybe it's better for you to make a pull request then we'll comment there.
